clusterloader2/testing/density/rc.yaml

```diff
@@ -10,7 +12,9 @@ spec:
     metadata:
       labels:
         name: {{.Name}}
+{{if .PodKind}}
```

Comment:
No ifs please. Seriously - this is making this unreadable.

Reply:
We should reuse the same label. Maybe sth like:
group: {{.Group}}

[where group is saturation or latency]?

Reply:
`if` removed.